src/login/config.js

```diff
@@ -0,0 +1,3 @@
+export const clientId = '147813579094-8qbv6canv6167qlga4k9788dg0qmgokn.apps.googleusercontent.com'
+
+export const cookiePolicy = 'single_host_origin'
```

Comment:
Should we move all of those into our settings.json file?

Reply:
Done

Reply:
This one is not done.